### PR TITLE
Last sprint's deliverables

### PR DESCRIPTION
**Pull request by ultiwinter**
Last sprint's deliverables
- User documentation
- Design documentation
- Build documentation
- Planning documentation
- Added conclusion and updated classifier comparison

### CODE REVIEW

File: Documentation/Classifier-Comparison.md [hunk 1]
```diff
@@ -141,3 +135,7 @@ In the following table we can see the F1-score of each model for each class in t
...
 The AdaBoost Classifier, trained on subset 1, performs best for the XL class. The KNN classifier got a slight boost in performance for the {S, M, L} and XL classes when using subset 1. All other models perform worse on subset 1.
+
+# Conclusion
+
+In summary, XGBoost consistently demonstrated superior performance, showcasing robust results across various splits and subsets. However, it is crucial to note that its elevated score is attributed to potential overfitting on the XS class. Given SumUp's emphasis on accurate predictions for higher classes, we recommend considering LightGBM. This model outperformed XGBoost in predicting the XL class and the other classes, offering better results in both the five-class and three-class splits.
```
Comment (Tims777) [inline Documentation/Classifier-Comparison.md:141]: Nice summary.

**Review by Tims777 — CHANGES_REQUESTED** [review]
Good work overall. Please just provide me with the source files of the compiled PDFs, because I would like to update the underlying Wiki.

File: Deliverables/sprint-13/build-documentation.pdf
[the remark this answers is not in the document]
Comment (Tims777) [reply Deliverables/sprint-13/build-documentation.pdf:1]: Done.

File: Deliverables/sprint-13/planning-documents.pdf
[the remark this answers is not in the document]
Comment (Tims777) [reply Deliverables/sprint-13/planning-documents.pdf:1]: Done.

File: Deliverables/sprint-13/build-documentation.pdf
[the remark this answers is not in the document]
Comment (Tims777) [reply Deliverables/sprint-13/build-documentation.pdf:1]: Felix created an updated version, which I just uploaded.

File: Deliverables/sprint-13/user-documentation.pdf
[the remark this answers is not in the document]
Comment (Tims777) [reply Deliverables/sprint-13/user-documentation.pdf:1]: Using the files you provided, I was able to do some minor adjustments. Thanks!

File: Deliverables/sprint-13/design-documentation.pdf
[the remark this answers is not in the document]
Comment (Tims777) [reply Deliverables/sprint-13/design-documentation.pdf:1]: I merged Architecture.md into the design documentation, so we do now also have the Component Diagram in there.

**Review by Tims777 — APPROVED** [review]

File: Documentation/User-Documentation.md
Comment (ur-tech) [inline Documentation/User-Documentation.md:1]: Very nice overview

**Review by ur-tech — APPROVED** [review]

**Pull request merged**